$OpenBSD: patch-pics_CMakeLists_txt,v 1.2 2013/07/05 14:46:05 zhuk Exp $
De-conflict with KDE3. Developer tool, not used in builds.
--- pics/CMakeLists.txt.orig	Fri Jul  5 17:42:38 2013
+++ pics/CMakeLists.txt	Fri Jul  5 18:41:31 2013
@@ -6,6 +6,7 @@ add_subdirectory( emoticons )
 set(ksvgtopng_SRCS ksvgtopng.cpp )
 kde4_add_executable(ksvgtopng NOGUI ${ksvgtopng_SRCS})
 target_link_libraries(ksvgtopng ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTSVG_LIBRARY})
+set_target_properties(ksvgtopng PROPERTIES OUTPUT_NAME ksvgtopng4)
 install(TARGETS ksvgtopng ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
 # create a symlink for the default icon theme,
